__test__/storage-events.test.ts

```diff
@@ -85,11 +85,12 @@ describe('Storage events', () => {
 
     ls.clearEvents();
     mockStorageEvent('app_logged_out');
-    expect(mockFn).not.toHaveBeenCalled();
+    expect(mockFn2).not.toHaveBeenCalled();
     jest.resetAllMocks();
 
     mockStorageEvent('app_logged_in');
-    expect(mockFn2).not.toHaveBeenCalled();
+    expect(mockFn).not.toHaveBeenCalled();
+    console.log(ls.events.listeners());
```

Comment:
@rishabhrpg 

Don't leave `console` statement in code

Reply:
Removed console logs